README.md

```diff
@@ -1,28 +1,28 @@
 <br>
 
 <p align="center">
-    <img width="300" src="/img/icon.png" alt="SurrealDB Icon">
+    <img width="300" src="./img/icon.png" alt="SurrealDB Icon">
 </p>
 
 <br>
 
-<p align="center">
+<!-- <p align="center">
     <a href="https://surrealdb.com#gh-dark-mode-only" target="_blank">
         <img width="300" src="/img/white/logo.svg" alt="SurrealDB Logo">
     </a>
     <a href="https://surrealdb.com#gh-light-mode-only" target="_blank">
         <img width="300" src="/img/black/logo.svg" alt="SurrealDB Logo">
     </a>
-</p>
+</p> -->
 
 <h3 align="center">
-    <a href="https://surrealdb.com#gh-dark-mode-only" target="_blank">
+    <!-- <a href="https://surrealdb.com#gh-dark-mode-only" target="_blank">
         <img src="/img/white/text.svg" height="15" alt="SurrealDB">
     </a>
     <a href="https://surrealdb.com#gh-light-mode-only" target="_blank">
         <img src="/img/black/text.svg" height="15" alt="SurrealDB">
-    </a>
-    is the ultimate cloud <br> database for tomorrow's applications
+    </a> -->
+    SurrealDB is the ultimate cloud <br> database for tomorrow's applications
```

Comment:
Here the SurrealDB logo is replaced by text.

Reply:
Removed the text, and restored into logo.